robot/client/client.go

```diff
@@ -786,6 +788,29 @@ func (rc *RobotClient) TransformPose(
 	return referenceframe.ProtobufToPoseInFrame(resp.Pose), nil
 }
 
+// TransformPointCloud will transform the pointcloud to the desired frame in the robot's frame system.
+// Do not move the robot between the generation of the initial pointcloud and the receipt
+// of the transformed pointcloud because that will make the transformations inaccurate.
+func (rc *RobotClient) TransformPointCloud(ctx context.Context, srcpc pointcloud.PointCloud, srcName, dstName string,
```

Comment:
Can this method use the frame service TransformPointCloud as well?

Reply:
It seems like a big burden to:
1. encode the entire pointcloud to PCD, 
2. send it over to the server TransformPCD,
3.  decode it back to PointCloud
4. Transform every point
5. encode back to PCD
6. send back to the client
7. decode back to PointCloud and return

In this way, only much more simple request is made, which are the 7 floats which represent the transformation to apply to every point in the pointcloud. Honestly, If we did https://viam.atlassian.net/browse/RSDK-1197, then we could directly inject the transform into the source pointcloud and be done!

Reply:
That makes sense, thanks for explaining!